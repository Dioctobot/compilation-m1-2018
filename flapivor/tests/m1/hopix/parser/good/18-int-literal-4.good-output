val x = -889275714
