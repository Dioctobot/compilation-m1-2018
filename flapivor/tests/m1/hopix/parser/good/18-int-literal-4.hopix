val x = 0xcafeBABE