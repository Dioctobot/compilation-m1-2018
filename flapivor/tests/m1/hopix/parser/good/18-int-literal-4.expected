val x = 3405691582